hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,92 @@
+defmodule HangmanTest do
```

Comment:
### Yes! tests!  +5

I love the fact you have tests.

You're missing a test for a repeated letter.

Also, if you added a function `Game.new(word)` to Hangman.Game it would have made these tests a bunch easier. :)